charts/devnet/templates/chains/cosmos/genesis.yaml

```diff
@@ -114,9 +114,35 @@ spec:
                 exit 0
               fi
 
-              echo "Running setup and config files..."
+              echo "Running setup genesis script..."
               bash -e /scripts/create-genesis.sh
               bash -e /scripts/update-genesis.sh
+          resources: {{- include "devnet.node.resources" ( dict "node" $chain "context" $ ) | trim | nindent 12 }}
+          volumeMounts:
+            - mountPath: {{ $chain.home }}
+              name: node
+            - mountPath: /configs
+              name: addresses
+            - mountPath: /scripts
+              name: scripts
+        - name: init-config
```

Comment:
not sure why this is needed, if we can avoid spinning up an extra init container, since the config update is perfored by the bash script, it is fully contained in that.

Reply:
IIRC it allowed me to debug and separate errors between genesis and config generation. Given that's the same image, I wouldn't expect to have a big delay in spinning up a new container.

Reply:
Ahh makes sense. Lets do this then